libs/utils/utils.js

```diff
@@ -1619,7 +1619,10 @@ function initSidekick() {
     const { default: init } = await import('./sidekick.js');
     const { getPreflightResults } = await import('../blocks/preflight/checks/preflightApi.js');
     init({ createTag, loadBlock, loadScript, loadStyle });
-    getPreflightResults(window.location.href, document);
+    getPreflightResults({
```

Comment:
can we simplify this? Why would we need to pass in globals.
I'd suggest just calling `getPreflightResults()`